### PR TITLE
Add node.js examples

### PR DESCRIPTION
**Pull request by firstred**
This PR adds the Payments API + Orders API node.js examples to the documentation

Requires:
- [ ]  https://github.com/mollie/mollie-api-node/pull/97 to be merged first

### CODE REVIEW

**Comment by firstred** [comment]
Fixed ruby examples (should stay unchanged) + squashed commits + rebase

File: source/reference/v2/customers-api/update-customer.rst [hunk 1]
```diff
@@ -116,6 +116,21 @@ Example
...
+   .. code-block:: javascript
+      :linenos:
+
+      const mollie = require('@mollie/api-client');
+      const mollieClient = mollie({ apiKey: 'test_dHar4XY7LxsDOtmnkVtjNVWXLSlXsM' });
+
+      (async () => {
+        const updatedCustomer = await mollie.customers.update({
+          customerId: 'cst_8wmqcHMN4U',
+
```
Comment (willemstuursma) [inline source/reference/v2/customers-api/update-customer.rst]: weird \n here

Comment (firstred) [reply source/reference/v2/customers-api/update-customer.rst]: Let's get rid of it then :slightly_smiling_face:

**Comment by firstred** [comment]
Might have te review the list subscription payments page again. Apparently, I had to add a code selector block before the JavaScript code block became visible. Not sure why the other languages were missing

**Pull request closed without merging**